README.md

```diff
@@ -54,6 +54,9 @@ On Windows:
 ```powershell
 curl.exe -Lo kind-windows-amd64.exe https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-windows-amd64
 Move-Item .\kind-windows-amd64.exe c:\some-dir-in-your-PATH\kind.exe
+
+# OR via Chocolatey
```

Comment:
maybe we could add a link to https://chocolatey.org/

Reply:
not: remove the `#`, this should not be an h1
Should also add this to the user guide.

Reply:
That should display as a powershell comment. Would you prefer it was separated from the code block for installing on Windows?

Reply:
🤦‍♂ I missed the ``` above, thanks